projects/sport-data-valley/manual_seeds/rheuma_responses.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require_relative 'helpers/people_helper'
+
+srand(123)
+
+START_DATE = Time.zone.parse('2021-11-24').beginning_of_day
```

Comment:
Can we do something dynamic? Like 3.weeks ago, and then if the end_date is 3.weeks ago + 3 weeks = today. So that when we run the script we always generate a bunch of recent responses? So that it looks good in the dashboard graphs.

Reply:
ok, i'll make it such that we don't get overlapped responses too